### ISSUE TITLE
[Feature Request] Display the album year, somewhere on the album details page.

### ISSUE THREAD

**Issue by Jieiku** [ampache/ampache#3228]
Display the album year, somewhere on the album details page.

## Describe what you don't like about Ampache, or what it is missing
Currently if you click into a an album, you see a list of the songs, nowhere on this page is the album year displayed.
I have to click edit album just to check what year the album is.

## Describe how you would like the feature to work**
Would really love it if the album year was displayed somewhere on the page.

**Comment by mitchray** [comment]
![firefox_18nBZDWDBH](https://user-images.githubusercontent.com/5735900/165002310-a99baa9d-ebaf-464d-a18a-abb0c16f0dc7.png)

Its showing in the album name for me?

**Comment by Jieiku** [comment]
hmmm, interesting indeed, here is mine on Firefox  (Ampache 5.2.1, master branch)

![2022-04-24_17-51-51](https://user-images.githubusercontent.com/106644/165004396-07230447-b7ef-4fde-a5f0-b8f820c7faf4.png)

And when I click Edit Album I can see the data is there:

![2022-04-24_17-52-18](https://user-images.githubusercontent.com/106644/165004418-9c5d7831-297b-4406-9b6a-560c9a031d12.png)

**Comment by mitchray** [comment]
Its from [here](https://github.com/ampache/ampache/blob/5691dce741429a47af6df7c0bac3c9b70fd95484/src/Repository/Model/Album.php#L676)

https://github.com/ampache/ampache/blob/5691dce741429a47af6df7c0bac3c9b70fd95484/src/Repository/Model/Album.php#L683

**Comment by Jieiku** [comment]
I see I wonder what you have in your ampache config that I do not, I would love it if it displayed the year as yours does.

**Comment by mitchray** [comment]
I'm not seeing any settings for this in the config file or settings, and you have it set so I'm stumped why it wouldn't show

Here are more details of my setup to cross-reference whats different.

I have these year-related tags in my files (either mostly OPUS with some MP3, which come from beets/picard which both use MusicBrainz as the source)

```
date
originaldate
originalyear
year
```

Changing it in the field updates it in the title

![firefox_fPfZmPXto7](https://user-images.githubusercontent.com/5735900/165017926-1e9541fe-61b1-4848-afc9-12211825f1af.png)

**Comment by troycarpenter** [comment]
I also see years in my album titles like @mitchray shows.  I don't see anything in my ampache config, but I have the following set in the GUI config:

Preferences -> Interface
Browse by Original Year for albums (falls back to Year) ON

In my tags, I only have the Date tag set with year (no day/month with it...just a year).  All songs from an album are tagged with the exact same data for album, artist, album_artist, date.  I think the genre and embedded album art can be different without ampache thinking it's a different album.  Actually artist can be different, too as long as album_artist is the same for all tracks.

**Comment by Jieiku** [comment]
New info, my collection has been the same for years. I removed an album from my directory and update the catalog.

```bash
sudo -u www-data php /var/www/ampache/bin/cli run:updateCatalog -a -g -i -c
```

I then put the file back and updated the catalog again.

That single album shows the year.

I think this might actually have to do with the way ampache stores the data in the database, maybe something changed at one point. No idea how I would get the rest of my collection to display the year on the albums, definitely dont want to start over on my ratings.

**Comment by mitchray** [comment]
Year is not part of the album title in the album art screen for mine...

![Screen Shot 2022-04-26 at 10 12 08 am](https://user-images.githubusercontent.com/5735900/165194360-bb2c1ebe-c648-4665-aa7d-7547d78f8e5c.png)

![Screen Shot 2022-04-26 at 10 12 21 am](https://user-images.githubusercontent.com/5735900/165194362-25ba9370-b19a-4ff6-af0b-ebf342930712.png)

**Comment by Jieiku** [comment]
Thanks @mitchray I am testing setting various date related id3 tags to see if maybe not all of them work the same.

**Comment by mitchray** [comment]
https://github.com/ampache/ampache/blob/5691dce741429a47af6df7c0bac3c9b70fd95484/src/Repository/Model/Album.php#L682-L684

I wonder if ```$this->year > 0``` is evaluating to false for some reason

**Comment by Jieiku** [comment]
ok, i checked that file @mitchray and noticed mine was not the same.

I tried a few days ago to update from 5.2.1 to 5.3.1 but ran into issues (I was on the master branch): https://github.com/ampache/ampache/issues/3223

At the time I rolled the container back to 5.2.1 when I could not resolve all of the issues. Today I tried updating to 5.3.1 a few times until I got it to work with no issues afterwards. The solution for me seems to have been switching from the master branch to the release5 branch.

After updating to 5.3.1 the year is now displayed on the album details page.
